specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10773 @@
+          "x-nullable": true
+        },
+        "functions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScoringFunction"
+          },
+          "description": "The collection of functions that influence the scoring of documents."
+        },
+        "functionAggregation": {
+          "$ref": "#/definitions/ScoringFunctionAggregation",
+          "description": "A value indicating how the results of individual scoring functions should be combined. Defaults to \"Sum\". Ignored if there are no scoring functions."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/rest/api/searchservice/Add-scoring-profiles-to-a-search-index"
+      },
+      "description": "Defines parameters for a search index that influence scoring in search queries."
+    },
+    "ScoringFunctionAggregation": {
+      "type": "string",
+      "enum": [
+        "sum",
+        "average",
+        "minimum",
+        "maximum",
+        "firstMatching"
+      ],
+      "x-ms-enum": {
+        "name": "ScoringFunctionAggregation",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "sum",
+            "name": "Sum",
+            "description": "Boost scores by the sum of all scoring function results."
+          },
+          {
+            "value": "average",
+            "name": "Average",
+            "description": "Boost scores by the average of all scoring function results."
+          },
+          {
+            "value": "minimum",
+            "name": "Minimum",
+            "description": "Boost scores by the minimum of all scoring function results."
+          },
+          {
+            "value": "maximum",
+            "name": "Maximum",
+            "description": "Boost scores by the maximum of all scoring function results."
+          },
+          {
+            "value": "firstMatching",
+            "name": "FirstMatching",
+            "description": "Boost scores using the first applicable scoring function in the scoring profile."
+          }
+        ]
+      },
+      "description": "Defines the aggregation function used to combine the results of all the scoring functions in a scoring profile."
+    },
+    "CorsOptions": {
+      "properties": {
+        "allowedOrigins": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of origins from which JavaScript code will be granted access to your index. Can contain a list of hosts of the form {protocol}://{fully-qualified-domain-name}[:{port#}], or a single '*' to allow all origins (not recommended)."
+        },
+        "maxAgeInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The duration for which browsers should cache CORS preflight responses. Defaults to 5 minutes
```

Comment:
@robertklee did you intend to make vectorIndexSize not required.

Reply:
@sazeesha098 it shows up as required on my side. The comment in the PR looks like it's pointing to CORS options; perhaps the version you reviewed was outdated? 

I intended to add it as "required" but since it's just the response deserialization object for index and service statistics, I think it doesn't matter whether it is required or not.